extension/options/history/historyView.js

```diff
@@ -0,0 +1,110 @@
+import { Database } from "./../../history.js";
```

Comment:
Whoops, you have the generated file in here. You should add `historyView.js` to `.gitignore` and delete it from the commit.

Reply:
Will do!